scene/gui/base_button.cpp

```diff
@@ -390,16 +391,26 @@ void BaseButton::shortcut_input(const Ref<InputEvent> &p_event) {
 	}
 }
 
-String BaseButton::get_tooltip(const Point2 &p_pos) const {
-	String tooltip = Control::get_tooltip(p_pos);
-	if (shortcut_in_tooltip && shortcut.is_valid() && shortcut->has_valid_event()) {
-		String text = shortcut->get_name() + " (" + shortcut->get_as_text() + ")";
-		if (!tooltip.is_empty() && shortcut->get_name().nocasecmp_to(tooltip) != 0) {
-			text += "\n" + atr(tooltip);
-		}
-		tooltip = text;
+Control *BaseButton::make_custom_tooltip(const String &p_text) const {
+	Control *control = Control::make_custom_tooltip(p_text);
+	if (control) {
+		return control;
+	}
+	if (!shortcut_in_tooltip || shortcut.is_null() || !shortcut->has_valid_event()) {
+		return nullptr; // Use the default tooltip.
+	}
+
+	String text = atr(shortcut->get_name()) + " (" + shortcut->get_as_text() + ")";
+	if (!p_text.is_empty() && shortcut->get_name().nocasecmp_to(p_text) != 0) {
```

Comment:
What's the `nocasecmp_to()` for? Why would these 2 strings be the equal?

Reply:
Yeah, added in #14913

Reply:
It does not make sense though. It skips displaying tooltip text if it's the same as shortcut name. But why would you even set them equal? I suspect it's some legacy thing from when shortcucts weren't displayed automatically.
I think this can be removed, but maybe in separate PR.